ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1246,3 +1246,52 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: api.ci
+  cron: '@daily'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  - base_ref: master
+    org: openshift
+    repo: ci-tools
+  - base_ref: master
+    org: kubernetes
+    repo: test-infra
+    workdir: true
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-prow-auto-testgrid-generator
+  spec:
+    containers:
+    - args:
+      - "--github-token-path=/etc/github/oauth"
+      - "--github-endpoint=https://api.github.com"
+      - "--github-graphql-endpoint=http://ghproxy/graphql"
+      - "--git-name=openshift-bot"
+      - "--git-email=openshift-bot@redhat.com"
+      - "--target-dir=."
+      - "--self-approve=false"
+      - "--assign=stevekuznetsov"
+      - "--testgrid-config=../../kubernetes/test-infra/config/testgrids/openshift"
+      - "--release-config=../../openshift/release/core-services/release-controller/_releases"
```

Comment:
You should be in th working directory for release anyway

Reply:
since i put the workdir: true for test-infra wouldn't I be in test-infra?